frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx

```diff
@@ -27,9 +27,28 @@ const fetchSubmissionsFromUser = async (submission_id: string) => {
     });
 };
 
+const editGrade = (submission: Submission, oldSubmission: Submission) => {
+  const submission_id = submission.submission_id;
+  const newGrade = submission.grading;
+
+  if (newGrade < 0 || newGrade > 20) {
+    return oldSubmission;
```

Comment:
Does this get caught anywhere so people know that the submission wasn't changed?

Reply:
What he said